solidity/contracts/bridge/Bridge.sol

```diff
@@ -432,6 +445,9 @@ contract Bridge is Ownable, EcdsaWalletOwner {
         redemptionTreasuryFeeDivisor = 2000; // 1/2000 == 5bps == 0.05% == 0.0005
         redemptionTxMaxFee = 1000; // 1000 satoshi
         redemptionTimeout = 172800; // 48 hours
+        movingFundsTxMaxTotalFee = 10000; // 10000 satoshi
```

Comment:
`redemptionTxMaxFee` is the max fee for the entire transaction. `depositTxMaxFee` is per deposit and `redemptionTxMaxFee` is per redemption. This is why `redemptionTxMaxFee` is much bigger.

Reply:
Right. What I am trying to say is that it *could happen* that we need to sweep a single deposit or that we need to process a single redemption request. Then, the fee per deposit and a fee per redemption will be equal to the fee of an entire Bitcoin transaction. And if we think 1k is not enough given we set `movingFundsTxMaxTotalFee` to 10k, it could happen that 1k will not be enough for `depositTxMaxFee` and `redemptionTxMaxFee`.

Reply:
Yeah but those are just initial values and there is a TODO about revisiting them before the launch. Does it work for now or do you want to change something?

Reply:
I would change them. I think they should be the same and if we agree on it, it is better to reflect it from the very beginning.

Is it a big change? Does it affect unit tests?